### PR TITLE
[DeadCode] RemoveDeadZeroAndOneOperationRector should not remove Constants

### PR DESCRIPTION
**Pull request by TwanVermeulen**
I'm curious about your thoughts! 
Sometimes we use constants that explain a certain number by words. 

If the constant number does not change the outcome of a mathematical expression, it's currently removed by `RemoveDeadZeroAndOneOperationRector`. So for constants that have a value of 1 this would be the case. 
I would prefer to keep the constant, since it's added to increase readability. 

What do you think? I added a test case that currently fails, but should pass imho.

Thanks for your input

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Hi, the constants should not be removed indeed :+1: 

Could you send fix as well? Thanks

**Comment by TwanVermeulen** [comment]
@TomasVotruba I added a fix. I do see some failed checks, but I'm not sure if they are related to my changes. Please let me know if there's any action required from my side.

**Comment by TwanVermeulen** [comment]
@TomasVotruba all checks are green after merging latest main

File: rules-tests/DeadCode/Rector/Plus/RemoveDeadZeroAndOneOperationRector/Fixture/skip_constants.php.inc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public function run()
+    {
+        $value = 5 * self::ONE;
+        $value = 5 / self::ONE;
+        $value = self::ONE * 5;
+        $value = self::ONE / 5;
+    }
+}
+
+?>
```
Comment (samsonasik) [inline rules-tests/DeadCode/Rector/Plus/RemoveDeadZeroAndOneOperationRector/Fixture/skip_constants.php.inc:37]: second part is unneeded for equal content

```suggestion
```

Comment (TomasVotruba) [reply rules-tests/DeadCode/Rector/Plus/RemoveDeadZeroAndOneOperationRector/Fixture/skip_constants.php.inc:37]: Indeed, I'll get this merged and fixup as the main work is done :+1:

**Comment by TomasVotruba** [comment]
@TwanVermeulen Thank you, this is good to go :+1: :slightly_smiling_face:

**Pull request merged**